sample-data.n3

```diff
@@ -665,99 +713,119 @@
         obo:RO_0000052          <http://vivo.mydomain.edu/individual/n1736> ;
         vivo:dateTimeInterval   <http://vivo.mydomain.edu/individual/n1750> ;
         vivo:roleContributesTo  <http://vivo.mydomain.edu/individual/n4085> .
 
 <http://vivo.mydomain.edu/individual/n476>
         a                foaf:Person ;
         rdfs:label       "Stevens, Emily K"@en-US ;
+        rdfs:label       "Emily K, Stevens"@fr-CA ;
         obo:ARG_2000028  <http://vivo.mydomain.edu/individual/n7188> ;
         vivo:relatedBy   <http://vivo.mydomain.edu/individual/n7123> .
 
 <http://vivo.mydomain.edu/individual/n4869>
         a                bibo:Conference ;
         rdfs:label       "17th Annual Conference on Philosophy and Rhetoric"@en-US ;
+        rdfs:label       "17ième  Conference Annuelle sur la Philosophie et la Rhetorique"@fr-CA ;
         obo:BFO_0000051  <http://vivo.mydomain.edu/individual/n7431> .
 
 <http://vivo.mydomain.edu/individual/n2022>
         a                      vivo:PresenterRole ;
         rdfs:label             "Keynote Speaker"@en-US ;
+        rdfs:label             "Présentateur"@fr-CA ;
         obo:BFO_0000054        <http://vivo.mydomain.edu/individual/n7431> ;
         obo:RO_0000052         <http://vivo.mydomain.edu/individual/n1736> ;
         vivo:dateTimeInterval  <http://vivo.mydomain.edu/individual/n4343> .
 
 <http://vivo.mydomain.edu/individual/n1116>
         a                    vcard:Address ;
         vcard:country        "USA"@en-US ;
+        vcard:country        "États Unis"@fr-CA ;
         vcard:locality       "Small City"@en-US ;
+        vcard:locality       "Small City"@fr-CA ;
         vcard:postalCode     "21626" ;
         vcard:region         "Kansas"@en-US ;
+        vcard:region         "Kansas"@fr-CA ;
+        vcard:streetAddress  "1400 Maple Street; "@fr-CA;
         vcard:streetAddress  "1400 Maple Street; "@en-US .
 
 <http://vivo.mydomain.edu/individual/n722>
         a                       vivo:EditorRole ;
         rdfs:label              "Associate Editor"@en-US ;
+        rdfs:label              "Editeur Associé"@fr-CA ;
         obo:RO_0000052          <http://vivo.mydomain.edu/individual/n1736> ;
         vivo:dateTimeInterval   <http://vivo.mydomain.edu/individual/n6945> ;
         vivo:roleContributesTo  <http://vivo.mydomain.edu/individual/n6748> .
 
 <http://vivo.mydomain.edu/individual/n5685>
         a                       vivo:EditorRole ;
         rdfs:label              "Editor in Chief"@en-US ;
+        rdfs:label              "Editeur en Chef"@fr-CA ;
         obo:RO_0000052          <http://vivo.mydomain.edu/individual/n1736> ;
         vivo:dateTimeInterval   <http://vivo.mydomain.edu/individual/n2585> ;
         vivo:roleContributesTo  <http://vivo.mydomain.edu/individual/n6748> .
 
 <http://vivo.mydomain.edu/individual/n6561>
         a                    skos:Concept ;
         rdfs:label           "Electracy"@en-US ;
```

Comment:
The english label for http://vivo.mydomain.edu/individual/n6561 "Electracy" (https://en.wikipedia.org/wiki/Electracy), which shouldn't be translated to "Électricité".

Reply:
Actually I was unable to translate that word in french that's why I wrote what google translate gave me. :)

Reply:
Maybe it's better to keep Electracy in French, too. Else I'm fine.